src/docs/development/data-and-backend/state-mgmt/options.md

```diff
@@ -35,6 +35,14 @@ A recommended approach.
 * [You might not need Redux: The Flutter edition][], by Ryan Edge
 * [Making sense of all those Flutter Providers][]
 
+## Riverpod
+
+A provider rewrite by the same author, compiled safe, more testable, more flexible
+
+* [Riverpod Homepage][]
+* [Riverpod Github][]
```

Comment:
This is redundant. From this page, we're linking to resources (like a homepage or an article or a video). You could link to pub.dev here, but since you're already linking to homepage above (which I think is the right choice), I suggest removing this line.

Alternatively, add a link to some tutorial (video or article).